framework/contracts/account/manager/Cargo.toml

```diff
@@ -55,6 +55,8 @@ abstract-app = { workspace = true, features = ["test-utils"] }
 abstract-testing = { workspace = true }
 abstract-macros = { workspace = true }
 
+cw20-base = { version = "1.1.0" }
```

Comment:
Can we use the workspace for this dependency?

Edit: I see that you depend on this version for the test. I think it would be better maintainability wise to have a macro like the `mock_app!` and `mock_adapter!` that lets us define standalone contracts easily and test with them without having to depend on an external lib (and its version). 

What do you think?

Reply:
> I think it would be better maintainability wise to have a macro like the `mock_app!` and `mock_adapter!` that lets us define standalone contracts easily and test with them without having to depend on an external lib (and its version).
> 
> What do you think?

Yeah we can add that as well, but that sounds like a rigged test to me.  Don't we want to make sure that we can install `external standalone contract`?
We can add `mock_app/mock_adapter` standalone tests in addition to this one.

Reply:
It felt like macros for standalone modules would be overkill, since we only care about cw2

Reply:
Sure, this should work for now :)